build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath gradlePlugins.androidTools
+        classpath "com.android.tools.build:gradle:3.6.3"
```

Comment:
Why have you changed your version here and not in `dependencies.gradle`? :thinking:

Reply:
Oh sorry that was Android Studio and I didn't notice it.
Also I forgot to check in dependencies.gradle, I fixed that too.